contracts/base/Dispatcher.sol

```diff
@@ -37,10 +37,10 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, Callbacks,
 
         success = true;
 
-        if (command < 0x20) {
-            if (command < 0x10) {
+        if (command < Commands.FOURTH_IF_BOUNDARY) {
```

Comment:
haha I find this less readable, but I guess more consistent since it's also constants.